### ISSUE TITLE
When opening the CDT blueprint on the Theia website the debug extension is active - but there's no gdb in PATH, which seems quite bad.

### ISSUE THREAD

**Issue by jonahgraham** [eclipse-cdt-cloud/cdt-cloud-blueprint#46]
> Note: when opening the CDT blueprint on the Theia website the debug extension is active - but there's no gdb in PATH, which seems quite bad.

_Originally posted by @GitMensch in https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/issues/33#issuecomment-1411005626_

**Comment by sdirix** [comment]
For now we mainly focused on the Electron version of CDT Cloud Blueprint. The Browser version still has more issues and the current state in the hosted version is not very good. We should think about removing or hiding that option until we're fully ready to present it and all open issues are properly documented.

**Comment by GitMensch** [comment]
Is there an online version of the Electron version of CDT Cloud Blueprint, too?

**Comment by sdirix** [comment]
The Electron version needs to run locally. You can either [download and install the latest release](https://www.eclipse.org/cdt-cloud/#blueprint) or start the development version from this repository via `yarn && yarn electron start`

**Comment by GitMensch** [comment]
Seems I've failed with that download link. After several hops I've got an Appimage from end of last year (https://download.eclipse.org/theia/cdt-cloud/latest/linux/CDTCloudBlueprint.AppImage), but that seems to be Theia based, too.

Note: the appimage does not contain libnss3, which had to be system-installed. This may be something to tackle (I don't know, other appimages I've used did not need it, only the minimal appimage libraries [fuse to not need to unpack it]).
